### ISSUE TITLE
Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65

### ISSUE THREAD

**Issue by cpt-buzz** [appium/appium#8944]
## The problem

Not able to launch my app on real device. It works fine on simulators. I get the above error every time.

## Environment

* Appium version (or git revision) that exhibits the issue:1.6.5
* Desktop OS/version used to run Appium: MacOS 10.12.5
* Real device: iPhone6S, iOS9.3.5

## Link to Appium logs

https://gist.github.com/cpt-buzz/3520a8e4f2517343b65e679abfd54414

**Comment by imurchie** [comment]
Can you turn on xcode logging (setting the `showXcodeLog` capability to `true`) and repost the logs?

**Comment by nrs16** [comment]
## **Problem**
I having the same issue with iOS 10.3.3.
## **Appium logs with showXcodeLog set to true**

The server is running
Start New SessionStop Server
[Appium] Welcome to Appium v1.6.5
[Appium] Appium REST http interface listener started on 0.0.0.0:4723
[HTTP] --> GET /wd/hub/sessions {}
[MJSONWP] Calling AppiumDriver.getSessions() with args: []
[MJSONWP] Responding to client with driver.getSessions() result: []
[HTTP] <-- GET /wd/hub/sessions 200 10 ms - 40 
[HTTP] --> POST /wd/hub/session {"desiredCapabilities":{"app":"/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app","automationName":"XCUITest","deviceName":"iPhone","fullReset":false,"nativeInstrumentsLib":true,"noReset":true,"platformName":"iOS","showXcodeLog":true,"udid":"87c9e5daab3df33577b91986b67f896da649988d","useNewWDA":false,"usePrebuiltWDA":true,"newCommandTimeout":0,"connectHardwareKeyboard":true}}
[MJSONWP] Calling AppiumDriver.createSession() with args: [{"app":"/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app","automationName":"XCUITest","deviceName":"iPhone","fullReset":false,"nativeInstrumentsLib":true,"noReset":true,"platformName":"iOS","showXcodeLog":true,"udid":"87c9e5daab3df33577b91986b67f896da649988d","useNewWDA":false,"usePrebuiltWDA":true,"newCommandTimeout":0,"connectHardwareKeyboard":true},null,null]
[BaseDriver] Event 'newSessionRequested' logged at 1501511878142 (17:37:58 GMT+0300 (EEST))
[Appium] Creating new XCUITestDriver (v2.30.0) session
[Appium] Capabilities:
[Appium]   app: '/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app'
[Appium]   automationName: 'XCUITest'
[Appium]   deviceName: 'iPhone'
[Appium]   fullReset: false
[Appium]   nativeInstrumentsLib: true
[Appium]   noReset: true
[Appium]   platformName: 'iOS'
[Appium]   showXcodeLog: true
[Appium]   udid: '87c9e5daab3df33577b91986b67f896da649988d'
[Appium]   useNewWDA: false
[Appium]   usePrebuiltWDA: true
[Appium]   newCommandTimeout: 0
[Appium]   connectHardwareKeyboard: true
[XCUITest] XCUITestDriver version: 2.30.0
[BaseDriver] Session created with session id: df26d4dd-1674-4e5d-b63e-51d6d8850e02
[XCUITest] Xcode version set to '8.3.2' (tools v8.3.0.0.1.1490382677)
[XCUITest] iOS SDK Version set to '10.3'
[BaseDriver] Event 'xcodeDetailsRetrieved' logged at 1501511878314 (17:37:58 GMT+0300 (EEST))
[XCUITest] Available devices: 87c9e5daab3df33577b91986b67f896da649988d
[XCUITest] Creating iDevice object with udid '87c9e5daab3df33577b91986b67f896da649988d'
[XCUITest] Determining device to run tests on: udid: '87c9e5daab3df33577b91986b67f896da649988d', real device: true
[BaseDriver] Using local app '/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app'
[BaseDriver] Event 'appConfigured' logged at 1501511878437 (17:37:58 GMT+0300 (EEST))
[XCUITest] Checking whether app '/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app' is actually present on file system
[XCUITest] App is present
[iOS] Getting bundle ID from app '/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app': 'com.alizzislamic.v3test'
[BaseDriver] Event 'resetStarted' logged at 1501511878442 (17:37:58 GMT+0300 (EEST))
[XCUITest] Reset: fullReset not set. Leaving as is
[BaseDriver] Event 'resetComplete' logged at 1501511878443 (17:37:58 GMT+0300 (EEST))
[iOSLog] Attempting iOS device log capture via libimobiledevice idevicesyslog
[iOSLog] Found idevicesyslog: '/usr/local/bin/idevicesyslog'
[BaseDriver] Event 'logCaptureStarted' logged at 1501511878586 (17:37:58 GMT+0300 (EEST))
[XCUITest] Setting up real device
[XCUITest] Installing '/Users/Ubanquity/Ubanquity.xcarchive/Products/Applications/ubq.app' on device with UUID '87c9e5daab3df33577b91986b67f896da649988d'...
[XCUITest] The app has been installed successfully.
[BaseDriver] Event 'appInstalled' logged at 1501511886329 (17:38:06 GMT+0300 (EEST))
[BaseDriver] Event 'wdaStartAttempted' logged at 1501511886332 (17:38:06 GMT+0300 (EEST))
[XCUITest] Using WDA path: '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent'
[XCUITest] Using WDA agent: '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj'
[XCUITest] Launching WebDriverAgent on the device
[XCUITest] Carthage found: '/usr/local/bin/carthage'
[XCUITest] Killing hanging processes
[XCUITest] Cannot parse major and minor version numbers from platformVersion "undefined". Will build for the default platform instead
[XCUITest] Beginning test with command 'xcodebuild test-without-building -project /Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj -scheme WebDriverAgentRunner -destination id=87c9e5daab3df33577b91986b67f896da649988d -configuration Debug' in directory '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent'
[XCUITest] Output from xcodebuild will be logged
[XCUITest] Starting iproxy to forward traffic from local port 8100 to device port 8100 over USB
[XCUITest] Log file for xcodebuild test: /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/E4A99D7D-46CF-4F9B-88C8-1721697A82E3/Session-WebDriverAgentRunner-2017-07-31_173812-ClCFmc.log
[Xcode] 2017-07-31 17:38:12.565 xcodebuild[13044:453639]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
[Xcode] /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/E4A99D7D-46CF-4F9B-88C8-1721697A82E3/Session-WebDriverAgentRunner-2017-07-31_173812-ClCFmc.log
[Xcode] 
[XCUITest] Waiting up to 60000ms for WebDriverAgent to start
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[Xcode] 2017-07-31 17:38:12.566 xcodebuild[13044:453632] [MT] IDETestOperationsObserverDebug: (565C97A9-36F6-4AF3-82C2-ACF9EDD827DA) Beginning test session WebDriverAgentRunner-565C97A9-36F6-4AF3-82C2-ACF9EDD827DA at 2017-07-31 17:38:12.565 with Xcode 8E2002 on target  {
[Xcode] 		deviceSerialNumber:         F4RP998ZG5MP
[Xcode] 		identifier:                 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		deviceClass:                iPhone
[Xcode] 		deviceName:                 iPhone
[Xcode] 		deviceIdentifier:           87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		productVersion:             10.2.1
[Xcode] 		buildVersion:               14D27
[Xcode] 		deviceSoftwareVersion:      10.2.1 (14D27)
[Xcode] 		deviceArchitecture:         arm64
[Xcode] 		deviceTotalCapacity:        11896262656
[Xcode] 		deviceAvailableCapacity:    8712896512
[Xcode] 		deviceIsTransient:          NO
[Xcode] 		ignored:                    NO
[Xcode] 		deviceIsBusy:               NO
[Xcode] 		deviceIsActivated:          YES
[Xcode] 		deviceActivationState:      Activated
[Xcode] 		isPasscodeLocked:           NO
[Xcode] 		deviceType:                 
[Xcode] 		supportedDeviceFamilies:    (
[Xcode]     1
[Xcode] )
[Xcode] 		applications:              (null)
[Xcode] 		provisioningProfiles:      (null)
[Xcode] 		activityProgress:          -2
[Xcode] 		activityTitle:             
[Xcode] 		hasInternalSupport:        NO
[Xcode] 		isSupportedOS:             YES
[Xcode] 		developerDiskMountError:   (null)
[Xcode] (null)
[Xcode] 	bootArgs:                  
[Xcode] 		} (10.2.1 (14D27))
[Xcode] 
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[Xcode] writeDictToFile:1278 ==== Successfully wrote Manifest cache to /var/folders/26/y8njkdm53b3gd6cvm6nf_dyh0000gp/C/com.apple.DeveloperTools/All/Xcode/EmbeddedAppDeltas/071228b0c740ddeba6b826a7d4b5e882/87c9e5daab3df33577b91986b67f896da649988d/ManifestCache.plist
[Xcode] 
[Xcode] 2017-07-31 17:38:14.103 xcodebuild[13044:453632] Error Domain=com.apple.dtdevicekit Code=-402620392 "Unable to install "WebDriverAgentRunner-Runner"" UserInfo={NSLocalizedDescription=Unable to install "WebDriverAgentRunner-Runner", NSLocalizedRecoverySuggestion=The certificate used to sign "WebDriverAgentRunner-Runner" has either expired or has been revoked. An updated certificate is required to sign and install the application.}
[Xcode] 2017-07-31 17:38:14.103 xcodebuild[13044:453632] Error Domain=IDETestOperationsObserverErrorDomain Code=6 "Early unexpected exit, operation never finished bootstrapping - no restart will be attempted" UserInfo={NSLocalizedDescription=Early unexpected exit, operation never finished bootstrapping - no restart will be attempted}
[Xcode] 
[Xcode] ** TEST EXECUTE FAILED **
[Xcode] 
[Xcode] 
[XCUITest] xcodebuild exited with code '65' and signal 'null'
[Xcode] Contents of xcodebuild log file '/Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/E4A99D7D-46CF-4F9B-88C8-1721697A82E3/Session-WebDriverAgentRunner-2017-07-31_173812-ClCFmc.log':
[Xcode] 17:38:12.566 xcodebuild[13044:453632] Beginning test session WebDriverAgentRunner-565C97A9-36F6-4AF3-82C2-ACF9EDD827DA at 2017-07-31 17:38:12.565 with Xcode 8E2002 on target  {
[Xcode] 		deviceSerialNumber:         F4RP998ZG5MP
[Xcode] 		identifier:                 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		deviceClass:                iPhone
[Xcode] 		deviceName:                 iPhone
[Xcode] 		deviceIdentifier:           87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		productVersion:             10.2.1
[Xcode] 		buildVersion:               14D27
[Xcode] 		deviceSoftwareVersion:      10.2.1 (14D27)
[Xcode] 		deviceArchitecture:         arm64
[Xcode] 		deviceTotalCapacity:        11896262656
[Xcode] 		deviceAvailableCapacity:    8712896512
[Xcode] 		deviceIsTransient:          NO
[Xcode] 		ignored:                    NO
[Xcode] 		deviceIsBusy:               NO
[Xcode] 		deviceIsActivated:          YES
[Xcode] 		deviceActivationState:      Activated
[Xcode] 		isPasscodeLocked:           NO
[Xcode] 		deviceType:                 
[Xcode] 		supportedDeviceFamilies:    (
[Xcode]     1
[Xcode] )
[Xcode] 		applications:              (null)
[Xcode] 		provisioningProfiles:      (null)
[Xcode] 		activityProgress:          -2
[Xcode] 		activityTitle:             
[Xcode] 		hasInternalSupport:        NO
[Xcode] 		isSupportedOS:             YES
[Xcode] 		developerDiskMountError:   (null)
[Xcode] (null)
[Xcode] 	bootArgs:                  
[Xcode] 		} (10.2.1 (14D27))
[Xcode] 17:38:12.566 xcodebuild[13044:453632] 	/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild
[Xcode] test-without-building
[Xcode] -project
[Xcode] /Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj
[Xcode] -scheme
[Xcode] WebDriverAgentRunner
[Xcode] -destination
[Xcode] id=87c9e5daab3df33577b91986b67f896da649988d
[Xcode] -configuration
[Xcode] Debug
[Xcode] 17:38:12.566 xcodebuild[13044:453632] Launching with Xcode.IDEFoundation.Launcher.PosixSpawn
[Xcode] 17:38:12.573 xcodebuild[13044:453636] Connecting to mobile device: 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 17:38:12.575 xcodebuild[13044:453636] Starting session with mobile device
[Xcode] 17:38:12.582 xcodebuild[13044:453632] Launch session started.
[Xcode] 17:38:12.629 xcodebuild[13044:453636] Starting test manager service on mobile device.
[Xcode] 17:38:12.635 xcodebuild[13044:453704] Received connection for test runner session
[Xcode] 17:38:12.636 xcodebuild[13044:453704] Listening for proxy connection request from the test bundle
[Xcode] 17:38:12.636 xcodebuild[13044:453632] Test runner session acquired connection.
[Xcode] 17:38:12.636 xcodebuild[13044:453632] Waiting for test process to launch.
[Xcode] 17:38:12.636 xcodebuild[13044:453704] Initiating session with identifier: 565C97A9-36F6-4AF3-82C2-ACF9EDD827DA
[Xcode] 17:38:12.693 xcodebuild[13044:453708] Got reply to session initiation request with identifier 565C97A9-36F6-4AF3-82C2-ACF9EDD827DA (result:error): 22: (null)
[Xcode] 17:38:12.793 xcodebuild[13044:453706] Installing com.apple.test.WebDriverAgentRunner-Runner using MobileInstall
[Xcode] 17:38:12.962 xcodebuild[13044:453635] app uninstall set - {(
[Xcode] )}
[Xcode] 17:38:12.962 xcodebuild[13044:453635] app install set - {(
[Xcode]     
[Xcode] )}
[Xcode] 17:38:13.813 xcodebuild[13044:453635] installApplicationBundleAtPath:withOptions:andError:withCallback: failed with {
[Xcode]     NSLocalizedDescription = "Unable to install \"WebDriverAgentRunner-Runner\"";
[Xcode]     NSLocalizedRecoverySuggestion = "The certificate used to sign \"WebDriverAgentRunner-Runner\" has either expired or has been revoked. An updated certificate is required to sign and install the application.";
[Xcode] }
[Xcode] 17:38:14.100 xcodebuild[13044:453706] Failed to install /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Build/Products/Debug-iphoneos/WebDriverAgentRunner-Runner.app - error Error Domain=com.apple.dtdevicekit Code=-402620392 "Unable to install "WebDriverAgentRunner-Runner"" UserInfo={NSLocalizedDescription=Unable to install "WebDriverAgentRunner-Runner", NSLocalizedRecoverySuggestion=The certificate used to sign "WebDriverAgentRunner-Runner" has either expired or has been revoked. An updated certificate is required to sign and install the application.}
[Xcode] 17:38:14.102 xcodebuild[13044:453632] Test operation failure: Launch session expired before checking in.
[Xcode] 17:38:14.102 xcodebuild[13044:453632] _finishWithError:Error Domain=IDETestOperationsObserverErrorDomain Code=5 "Launch session expired before checking in." UserInfo={NSLocalizedDescription=Launch session expired before checking in.} didCancel: 1
[Xcode] 
[BaseDriver] Event 'wdaStartFailed' logged at 1501511894139 (17:38:14 GMT+0300 (EEST))
[XCUITest] Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65
[XCUITest] Quitting and uninstalling WebDriverAgent, then retrying
[XCUITest] Shutting down sub-processes
[XCUITest] Shutting down iproxy process (pid 13043)
[XCUITest] iproxy exited with code 'null'
[XCUITest] Removing WDA application from device
[BaseDriver] Event 'wdaStartAttempted' logged at 1501511904325 (17:38:24 GMT+0300 (EEST))
[XCUITest] Using WDA path: '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent'
[XCUITest] Using WDA agent: '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj'
[XCUITest] Launching WebDriverAgent on the device
[XCUITest] Carthage found: '/usr/local/bin/carthage'
[XCUITest] Killing hanging processes
[XCUITest] Cannot parse major and minor version numbers from platformVersion "undefined". Will build for the default platform instead
[XCUITest] Beginning test with command 'xcodebuild test-without-building -project /Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj -scheme WebDriverAgentRunner -destination id=87c9e5daab3df33577b91986b67f896da649988d -configuration Debug' in directory '/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent'
[XCUITest] Output from xcodebuild will be logged
[XCUITest] Starting iproxy to forward traffic from local port 8100 to device port 8100 over USB
[XCUITest] Log file for xcodebuild test: /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/EAE689C6-077B-48CE-97B6-3E2ED551F266/Session-WebDriverAgentRunner-2017-07-31_173830-2jGhFQ.log
[Xcode] 2017-07-31 17:38:30.520 xcodebuild[13064:454136]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
[Xcode] /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/EAE689C6-077B-48CE-97B6-3E2ED551F266/Session-WebDriverAgentRunner-2017-07-31_173830-2jGhFQ.log
[Xcode] 
[XCUITest] Waiting up to 60000ms for WebDriverAgent to start
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[Xcode] 2017-07-31 17:38:30.520 xcodebuild[13064:454060] [MT] IDETestOperationsObserverDebug: (932C6AEC-5943-4F4B-A2F1-E580BE855AE6) Beginning test session WebDriverAgentRunner-932C6AEC-5943-4F4B-A2F1-E580BE855AE6 at 2017-07-31 17:38:30.520 with Xcode 8E2002 on target  {
[Xcode] 		deviceSerialNumber:         F4RP998ZG5MP
[Xcode] 		identifier:                 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		deviceClass:                iPhone
[Xcode] 		deviceName:                 iPhone
[Xcode] 		deviceIdentifier:           87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		productVersion:             10.2.1
[Xcode] 		buildVersion:               14D27
[Xcode] 		deviceSoftwareVersion:      10.2.1 (14D27)
[Xcode] 		deviceArchitecture:         arm64
[Xcode] 		deviceTotalCapacity:        11896262656
[Xcode] 		deviceAvailableCapacity:    8712732672
[Xcode] 		deviceIsTransient:          NO
[Xcode] 		ignored:                    NO
[Xcode] 		deviceIsBusy:               NO
[Xcode] 		deviceIsActivated:          YES
[Xcode] 		deviceActivationState:      Activated
[Xcode] 		isPasscodeLocked:           NO
[Xcode] 		deviceType:                 
[Xcode] 		supportedDeviceFamilies:    (
[Xcode]     1
[Xcode] )
[Xcode] 		applications:              (null)
[Xcode] 		provisioningProfiles:      (null)
[Xcode] 		activityProgress:          -2
[Xcode] 		activityTitle:             
[Xcode] 		hasInternalSupport:        NO
[Xcode] 		isSupportedOS:             YES
[Xcode] 		developerDiskMountError:   (null)
[Xcode] (null)
[Xcode] 	bootArgs:                  
[Xcode] 		} (10.2.1 (14D27))
[Xcode] 
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[JSONWP Proxy] Proxying [GET /status] to [GET http://localhost:8100/status] with no body
[Xcode] writeDictToFile:1278 ==== Successfully wrote Manifest cache to /var/folders/26/y8njkdm53b3gd6cvm6nf_dyh0000gp/C/com.apple.DeveloperTools/All/Xcode/EmbeddedAppDeltas/071228b0c740ddeba6b826a7d4b5e882/87c9e5daab3df33577b91986b67f896da649988d/ManifestCache.plist
[Xcode] 
[Xcode] 2017-07-31 17:38:32.001 xcodebuild[13064:454060] Error Domain=com.apple.dtdevicekit Code=-402620392 "Unable to install "WebDriverAgentRunner-Runner"" UserInfo={NSLocalizedDescription=Unable to install "WebDriverAgentRunner-Runner", NSLocalizedRecoverySuggestion=The certificate used to sign "WebDriverAgentRunner-Runner" has either expired or has been revoked. An updated certificate is required to sign and install the application.}
[Xcode] 2017-07-31 17:38:32.002 xcodebuild[13064:454060] Error Domain=IDETestOperationsObserverErrorDomain Code=6 "Early unexpected exit, operation never finished bootstrapping - no restart will be attempted" UserInfo={NSLocalizedDescription=Early unexpected exit, operation never finished bootstrapping - no restart will be attempted}
[Xcode] 
[Xcode] ** TEST EXECUTE FAILED **
[Xcode] 
[Xcode] 
[XCUITest] xcodebuild exited with code '65' and signal 'null'
[Xcode] Contents of xcodebuild log file '/Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Logs/Test/EAE689C6-077B-48CE-97B6-3E2ED551F266/Session-WebDriverAgentRunner-2017-07-31_173830-2jGhFQ.log':
[Xcode] 17:38:30.521 xcodebuild[13064:454060] Beginning test session WebDriverAgentRunner-932C6AEC-5943-4F4B-A2F1-E580BE855AE6 at 2017-07-31 17:38:30.520 with Xcode 8E2002 on target  {
[Xcode] 		deviceSerialNumber:         F4RP998ZG5MP
[Xcode] 		identifier:                 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		deviceClass:                iPhone
[Xcode] 		deviceName:                 iPhone
[Xcode] 		deviceIdentifier:           87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 		productVersion:             10.2.1
[Xcode] 		buildVersion:               14D27
[Xcode] 		deviceSoftwareVersion:      10.2.1 (14D27)
[Xcode] 		deviceArchitecture:         arm64
[Xcode] 		deviceTotalCapacity:        11896262656
[Xcode] 		deviceAvailableCapacity:    8712732672
[Xcode] 		deviceIsTransient:          NO
[Xcode] 		ignored:                    NO
[Xcode] 		deviceIsBusy:               NO
[Xcode] 		deviceIsActivated:          YES
[Xcode] 		deviceActivationState:      Activated
[Xcode] 		isPasscodeLocked:           NO
[Xcode] 		deviceType:                 
[Xcode] 		supportedDeviceFamilies:    (
[Xcode]     1
[Xcode] )
[Xcode] 		applications:              (null)
[Xcode] 		provisioningProfiles:      (null)
[Xcode] 		activityProgress:          -2
[Xcode] 		activityTitle:             
[Xcode] 		hasInternalSupport:        NO
[Xcode] 		isSupportedOS:             YES
[Xcode] 		developerDiskMountError:   (null)
[Xcode] (null)
[Xcode] 	bootArgs:                  
[Xcode] 		} (10.2.1 (14D27))
[Xcode] 17:38:30.521 xcodebuild[13064:454060] 	/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild
[Xcode] test-without-building
[Xcode] -project
[Xcode] /Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgent.xcodeproj
[Xcode] -scheme
[Xcode] WebDriverAgentRunner
[Xcode] -destination
[Xcode] id=87c9e5daab3df33577b91986b67f896da649988d
[Xcode] -configuration
[Xcode] Debug
[Xcode] 17:38:30.521 xcodebuild[13064:454060] Launching with Xcode.IDEFoundation.Launcher.PosixSpawn
[Xcode] 17:38:30.527 xcodebuild[13064:454067] Connecting to mobile device: 87c9e5daab3df33577b91986b67f896da649988d
[Xcode] 17:38:30.530 xcodebuild[13064:454067] Starting session with mobile device
[Xcode] 17:38:30.541 xcodebuild[13064:454060] Launch session started.
[Xcode] 17:38:30.568 xcodebuild[13064:454067] Starting test manager service on mobile device.
[Xcode] 17:38:30.576 xcodebuild[13064:454146] Received connection for test runner session
[Xcode] 17:38:30.576 xcodebuild[13064:454146] Listening for proxy connection request from the test bundle
[Xcode] 17:38:30.576 xcodebuild[13064:454060] Test runner session acquired connection.
[Xcode] 17:38:30.576 xcodebuild[13064:454060] Waiting for test process to launch.
[Xcode] 17:38:30.577 xcodebuild[13064:454146] Initiating session with identifier: 932C6AEC-5943-4F4B-A2F1-E580BE855AE6
[Xcode] 17:38:30.616 xcodebuild[13064:454136] Got reply to session initiation request with identifier 932C6AEC-5943-4F4B-A2F1-E580BE855AE6 (result:error): 22: (null)
[Xcode] 17:38:30.711 xcodebuild[13064:454145] Installing com.apple.test.WebDriverAgentRunner-Runner using MobileInstall
[Xcode] 17:38:30.861 xcodebuild[13064:454136] app uninstall set - {(
[Xcode] )}
[Xcode] 17:38:30.861 xcodebuild[13064:454136] app install set - {(
[Xcode]     
[Xcode] )}
[Xcode] 17:38:31.725 xcodebuild[13064:454136] installApplicationBundleAtPath:withOptions:andError:withCallback: failed with {
[Xcode]     NSLocalizedDescription = "Unable to install \"WebDriverAgentRunner-Runner\"";
[Xcode]     NSLocalizedRecoverySuggestion = "The certificate used to sign \"WebDriverAgentRunner-Runner\" has either expired or has been revoked. An updated certificate is required to sign and install the application.";
[Xcode] }
[Xcode] 17:38:31.999 xcodebuild[13064:454145] Failed to install /Users/Ubanquity/Library/Developer/Xcode/DerivedData/WebDriverAgent-dikkwtrisltbeobjmfvpthwwekvs/Build/Products/Debug-iphoneos/WebDriverAgentRunner-Runner.app - error Error Domain=com.apple.dtdevicekit Code=-402620392 "Unable to install "WebDriverAgentRunner-Runner"" UserInfo={NSLocalizedDescription=Unable to install "WebDriverAgentRunner-Runner", NSLocalizedRecoverySuggestion=The certificate used to sign "WebDriverAgentRunner-Runner" has either expired or has been revoked. An updated certificate is required to sign and install the application.}
[Xcode] 17:38:32.001 xcodebuild[13064:454060] Test operation failure: Launch session expired before checking in.
[Xcode] 17:38:32.001 xcodebuild[13064:454060] _finishWithError:Error Domain=IDETestOperationsObserverErrorDomain Code=5 "Launch session expired before checking in." UserInfo={NSLocalizedDescription=Launch session expired before checking in.} didCancel: 1
[Xcode] 
[BaseDriver] Event 'wdaStartFailed' logged at 1501511912032 (17:38:32 GMT+0300 (EEST))
[XCUITest] Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65
[XCUITest] Quitting and uninstalling WebDriverAgent, then retrying
[XCUITest] Shutting down sub-processes
[XCUITest] Shutting down iproxy process (pid 13063)
[XCUITest] iproxy exited with code 'null'
[XCUITest] Removing WDA application from device
[XCUITest] Error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65
    at XCUITestDriver.quitAndUninstall$ (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/lib/driver.js:386:15)
    at tryCatch (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:136:37)
 Error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65
    at XCUITestDriver.quitAndUninstall$ (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/lib/driver.js:386:15)
    at tryCatch (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:136:37)
[XCUITest] Not clearing log files. Use `clearSystemFiles` capability to turn on.
[iOSLog] Stopping iOS log capture
[MJSONWP] Encountered internal error running command: Error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 65
    at XCUITestDriver.quitAndUninstall$ (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-xcuitest-driver/lib/driver.js:386:15)
    at tryCatch (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/babel-runtime/regenerator/runtime.js:136:37)
[HTTP] <-- POST /wd/hub/session 500 44107 ms - 230

**Comment by imurchie** [comment]
The logs seem pretty clear in this case:
```
The certificate used to sign "WebDriverAgentRunner-Runner" has either expired or has been revoked. An updated certificate is required to sign and install the application
```

You need to update your signing certificates.
